src/test/lib/ApiTest.cpp

```diff
@@ -615,6 +615,90 @@ ListenerFailSendResumeCallback(
 }
 #endif
 
+// static
```

Comment:
Can you remove all this commented out stuff?

Reply:
Draft PR might be miss prone...